lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -2946,9 +2946,13 @@ ParseResult FIRCircuitParser::parseModule(CircuitOp circuit,
 
         // If we got to the next module, then we're done.
       case FIRToken::kw_module:
-      case FIRToken::kw_extmodule:
-        return success();
-
+      case FIRToken::kw_extmodule: {
+        // All module declerations should have the same indentation
+        // level.  Use this fact to differentiate between module
+        // declarations and usages of "module" as identifiers.
+        if (getIndentation() == indent)
```

Comment:
Note, no action requested:
Presently, the test input with one less space before `module.in` is not rejected, and ignored entirely if drops below "module" level.  Both are something the module parser probably can/should reject (elsewhere, separate PR/effort), although could reject here I believe if less-than the indentation of the other modules?